scipy/stats/_continuous_distns.py

```diff
@@ -9247,6 +9247,49 @@ def expect(self, func=None, args=(), loc=0, scale=1, lb=None, ub=None,
         return super().expect(func, args, loc,
                               scale, lb, ub, conditional, **kwds)
 
+    @_call_super_mom
+    @extend_notes_in_docstring(rv_continuous, notes="""\
+        The `scale` parameter is ignored. Fit data must lie in the interval
+        [-pi, pi].\n\n""")
+    def fit(self, data, *args, **kwds):
+        if kwds.pop('superfit', False):
+            return super().fit(data, *args, **kwds)
+
+        data, fshape, floc, fscale = _check_fit_input_parameters(self, data,
+                                                                 args, kwds)
+        if self.a == -np.pi:
```

Comment:
Not sure this is the preferred way but it allows `vonmises_line` to be untouched by this PR.